orion/pkg/services/achieve_service.go

```diff
@@ -58,3 +68,13 @@ func (as *achieveService) Delete(id uint) error {
 	err := repos.AchieveRepo.Delete(id)
 	return err
 }
+
+func (as *achieveService) Publish(ids []uint) error {
+	for _, id := range ids {
+		err := repos.AchieveRepo.Publish(id)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
nice! Can you write a TODO commented statement somewhere here
```
// TODO: Use DB Transactions
```
I'll go over what it is later, but TLDR it's a way to make bulk updates/creates/deletes whatever correct in any situation.